@kiva/kv-tokens/dist/scss/_variables.scss

```diff
@@ -0,0 +1,172 @@
+$colors-theme-default-background-secondary: #F5F5F5;
+$colors-theme-default-background-tertiary: #E0E0E0;
+$colors-theme-default-background-action: #2B7C5F;
+$colors-theme-default-background-action-highlight: #277056;
+$colors-theme-default-background-danger: #E42A2A;
+$colors-theme-default-background-danger-highlight: #CE2626;
+$colors-theme-default-background-caution: #FEE16E;
+$colors-theme-default-border-primary: #212121;
+$colors-theme-default-border-primary-inverse: #FFFFFF;
+$colors-theme-default-border-secondary: #757575;
+$colors-theme-default-border-tertiary: #C4C4C4;
+$colors-theme-default-border-action: #2B7C5F;
+$colors-theme-default-border-action-highlight: #277056;
+$colors-theme-default-border-danger: #E42A2A;
+$colors-theme-default-border-danger-highlight: #CE2626;
+$colors-theme-dark-text-primary: #F5F5F5;
+$colors-theme-dark-text-primary-inverse: #212121;
+$colors-theme-dark-text-secondary: #9E9E9E;
+$colors-theme-dark-text-tertiary: #757575;
+$colors-theme-dark-text-action: #4AB67E;
+$colors-theme-dark-text-action-highlight: #6AC395;
+$colors-theme-dark-text-danger: #E42A2A;
+$colors-theme-dark-text-danger-highlight: #CE2626;
+$colors-theme-dark-background-primary: #000000;
+$colors-theme-dark-background-primary-inverse: #F5F5F5;
+$colors-theme-dark-background-secondary: #212121;
+$colors-theme-dark-background-tertiary: #454545;
+$colors-theme-dark-background-action: #4AB67E;
+$colors-theme-dark-background-action-highlight: #6AC395;
+$colors-theme-dark-background-danger: #E42A2A;
+$colors-theme-dark-background-danger-highlight: #CE2626;
+$colors-theme-dark-background-caution: #FEE16E;
+$colors-theme-dark-border-primary: #F5F5F5;
+$colors-theme-dark-border-primary-inverse: #212121;
+$colors-theme-dark-border-secondary: #9E9E9E;
+$colors-theme-dark-border-tertiary: #757575;
+$colors-theme-dark-border-action: #4AB67E;
+$colors-theme-dark-border-action-highlight: #6AC395;
+$colors-theme-dark-border-danger: #E42A2A;
+$colors-theme-dark-border-danger-highlight: #CE2626;
+$colors-theme-mint-text-secondary: #228752;
+$colors-theme-mint-text-tertiary: #28A062;
+$colors-theme-mint-background-primary: #95D4B3;
+$colors-theme-mint-border-tertiary: #4AB67E;
+$colors-theme-dark-green-text-primary: #F5F5F5;
+$colors-theme-dark-green-text-primary-inverse: #212121;
+$colors-theme-dark-green-text-secondary: #9E9E9E;
+$colors-theme-dark-green-background-primary: #2AA967;
+$colors-theme-dark-green-background-primary-inverse: #F5F5F5;
+$colors-theme-dark-green-background-secondary: #228752;
+$space-1: 8;
+$space-2: 16;
+$space-3: 24;
+$space-4: 32;
+$space-5: 40;
+$space-6: 48;
+$space-7: 56;
+$space-8: 64;
+$space-9: 72;
+$space-10: 80;
+$space-11: 88;
+$space-12: 96;
+$space-13: 104;
+$space-14: 112;
+$space-15: 120;
+$space-16: 128;
+$space-0-5: 4;
+$space-1-5: 12;
+$space-2-5: 20;
+$space-3-5: 28;
+$space-4-5: 36;
+$fonts-sans: PostGrotesk;
+$font-sizes-base-sm: 16;
+$font-sizes-base-md: 16;
+$font-sizes-base-lg: 17;
```

Comment:
All of these unitless values are kindof a pain to deal with in SASS and CSS. I wonder if we should add 'px' to them in the tokens file directly.

Reply:
Let's put px in tokens file.